README.md

```diff
@@ -60,9 +60,14 @@ A complete program with this code can be found in the examples folder.
 
 ## Limitations
 
-Currently we only have implementations for getting stack traces on x86_64 processors running
-Linux/Windows or OSX. We don't have the abilitiy to get stack traces at all from ARM or i686
-processors, or from FreeBSD.
+Currently we only have implementations for getting stack traces on some platforms:
+
+|         | Linux | Windows | OSX | FreeBSD |
+|---------|-------|---------|-----|---------|
+| i686    |       |         |     |         |
+| x86-64  | yes   | yes     | yes |         |
```

Comment:
Thanks for putting in this table - but the original info was incorrect, I removed the OSX implementation since it was very unstable =(
```suggestion
| x86-64  | yes   | yes     |     |         |
```

Reply:
Ah, makes sense.